services/graphql-server/src/graphql/definitions/platform/content/index.js

```diff
@@ -284,23 +287,71 @@ input ContentSitemapNewsUrlsQueryInput {
 }
 
 input AllPublishedContentQueryInput {
+  """
```

Comment:
Change all of these to inline strings please... i.e.`"Limits results to content with a primary site matching the current site context"`